### DISCUSSION TITLE
[TransferList] - Add prop to have `move all` only move items found by the search term

### DISCUSSION THREAD

**Discussion by justinsmid** [mantinedev/mantine#3430] [Feature requests]
I would like to have a prop added to TransferList that makes it so only found items get moved when you press the `move all` button.

E.g.: if a list has the following items:

- React
- Angular
- Next.js

And I search for `a` and press `move all`, only `React` and `Angular` would be moved since `Next.js` doesn't contain an `a`.

I would also be willing to implement it myself if there is no objection to adding this feature. Please let me know!

**Comment by rtivital** [comment]
You are welcome to send a PR. Note that if it breaks anything, it will be rolled back, so please verify that the component works correctly before submitting a PR.
